redwood-treehouse-lazylayout-uiview/build.gradle

```diff
@@ -20,3 +20,11 @@ kotlin {
     }
   }
 }
+
+// TODO: find a way to get Objective-C conflicting overloads to play nice with Kotlin metadata.
+// https://kotlinlang.slack.com/archives/C9JM6Q2UX/p1683686935270659
+tasks.configureEach { task ->
+  if (task.name == "compileCommonMainKotlinMetadata") {
+    task.onlyIf { false }
```

Comment:
We used to have to do this a long, long time ago in this repo (back when we used `includeBuild` to split the library + samples) and it caused tons of problems with publishing. This might be okay because it's a leaf module in terms of publishing (no other published module depends on it).

Reply:
https://github.com/cashapp/redwood/pull/1006/commits/1f92af9d605174af886dc9fbdb91c088346beae5